frontend/src/pages/404/index.jsx

```diff
@@ -1,10 +1,13 @@
 import { useTranslation } from 'react-i18next';
 import { Link } from 'react-router-dom';
+import { useTernaryDarkMode } from 'usehooks-ts';
 
 function NotFound() {
   const { t: tPNF } = useTranslation('translation', {
     keyPrefix: 'pageNotFound',
   });
+  const { isDarkMode } = useTernaryDarkMode();
+  const paragraphClass = `fs-5 mb-3 text-${isDarkMode ? 'white' : 'black'}`;
```

Comment:
@sergiozeppo do we have bootstrap themes? I think we can use bootstrap css variables/classes that changes color depends on theme